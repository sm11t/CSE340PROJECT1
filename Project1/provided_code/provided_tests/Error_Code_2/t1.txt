TASKS
1 2
POLY
    F1 = X + 2;
    F2(Y,Z) = x (x + 2)^2;
EXECUTE
    a = F1(2);
    OUTPUT a;
INPUTS
    3 5 6