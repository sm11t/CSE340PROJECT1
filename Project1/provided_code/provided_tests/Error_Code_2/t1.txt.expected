Semantic Error Code 2: 4 5 5 
